Source/santad/SNTPolicyProcessor.m

```diff
@@ -131,6 +135,14 @@ - (nonnull SNTCachedDecision *)decisionForFileInfo:(nonnull SNTFileInfo *)fileIn
     return cd;
   }
 
+  if (csInfoError && csInfoError.code != errSecCSUnsigned &&
```

Comment:
If we are considering this a "blacklisted scope", similar to the __PAGEZERO scope, shouldn't this go into the `fileIsScopeBlacklisted` method? Or at least before the `fileIsScopeWhitelisted` check?

Reply:
Good point, I've moved it above both `fileIsScopeBlacklisted`/`fileIsScopeWhitelisted`.